docs/index.rst

```diff
@@ -4,9 +4,10 @@
 build
 *****
 
-A simple, correct :pep:`517` build frontend.
+A simple, correct Python build frontend.
 
-build will invoke the :pep:`517` hooks to build a distribution package.
+build implements handling for pyproject.toml-based builds, invoking
+build-backend hooks as appropriate to build a distribution package.
```

Comment:
Is it possible to link to definitions of hooks and dist packages on pypa.io?

Reply:
Okay to do that in a followup?

Reply:
Yep.

Reply:
Lets merge this and do it in a followup.